File: Back-end/src/HorizonEvents.Domain/Speaker.cs
using System.Collections.Generic;

namespace HorizonEvents.Domain
{
    public class Speaker
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string SmallResume { get; set; }
        public string ImageURL { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public IEnumerable<SocialMedia> SocialMedias { get; set; }
        public IEnumerable<SpeakerEvent> SpeakerEvents { get; set; }

    }
}

File: Front-End/HorizonEvents-App/src/app/models/Speaker.ts
import { SocialMedia } from "./SocialMedia";

export interface Speaker{
 id: number;
 name: string;
 smallResume: string;
 imageURL: string;
 phoneNumber: string;
 email: string;
 socialMedias: SocialMedia[];
 speakerEvents: Event[];
}
